seedseeker/generators/ran3.py

```diff
@@ -0,0 +1,73 @@
+from typing import Iterator
+import random
+
+
+def ran3(seed: int):
```

Comment:
returns Generator[int]

Reply:
Sorry, actually it should be `IntegerRNG` from `defs.py`, I forgot I added the alias...